lms/djangoapps/hera/models.py

```diff
@@ -295,9 +295,9 @@ def delete(self, *args, **kwargs):
 
 
 class Mascot(models.Model):
-    user_dashboard = models.ImageField(upload_to='hera', null=True, blank=True)
-    user_dashboard_modal = models.ImageField(upload_to='hera', null=True, blank=True)
-    onboarding = models.ImageField(upload_to='hera', null=True, blank=True)
+    user_dashboard = models.ImageField(upload_to='hera/mascots', null=True, blank=True)
```

Comment:
is it worth making some constant for `upload_to` attribute not to repeat this string three times?